### PR TITLE
[NUI] Open newly added APIs in Button and Notification

### PR DESCRIPTION
**Pull request by rabbitfor**
Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### API Changes ###

 - ACR: https://code.sec.samsung.net/jira/browse/TCSACR-411

### CODE REVIEW

**Review by hyunjushin — APPROVED** [review]

File: src/Tizen.NUI.Components/Controls/Notification.cs [hunk 1]
```diff
@@ -165,7 +165,7 @@ private Timer Timer
         /// Notification.MakeToast("Hello World!", Notification.ToastBottom).Post(Notification.ToastShort);
```
Comment (wiertel) [inline src/Tizen.NUI.Components/Controls/Notification.cs:165]: line 163: "period time" -> "period of time"
You can consider changing the sentence a little more: "how to show a toast at the bottom of the screen for a short period of time".

File: src/Tizen.NUI.Components/Controls/Button.cs [hunk 2]
```diff
@@ -664,7 +664,7 @@ public LinearLayout.Alignment ItemAlignment
         /// The value is applied when there exist icon and text both.
```
Comment (wiertel) [inline src/Tizen.NUI.Components/Controls/Button.cs:664]:
```suggestion
        /// The value is applied when the button has both icon visual and text set.
```
I'm not sure about "icon visual". You can drop the word "visual".
BTW - how user sets the "icon"? There are properties for various visuals. Does one of them relate to icon?

File: src/Tizen.NUI.Components/Controls/Notification.cs (hunk 1 above)
Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/Notification.cs:165]: Done

File: src/Tizen.NUI.Components/Controls/Button.cs (hunk 2 above)
Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/Button.cs]: Done.
User can set icon using `iconURL` property.

**Review by wiertel — APPROVED** [review]

**Comment by dongsug-song** [comment]
https://code.sec.samsung.net/jira/browse/TCSACR-411 
has been accepted, so now be merged.

**Pull request merged**